src/ItemMeasurer.js

```diff
@@ -0,0 +1,107 @@
+// @flow
+
+import { Component } from 'react';
+import { findDOMNode } from 'react-dom';
```

Comment:
Does this mean dynamic measurements won't work with concurrent mode?

Reply:
If `findDOMNode` goes away in the future, I will have to switch to a ref-based approach that uses `cloneElement()` but I may leave that for future Brian to deal with.

Reply:
Meh...maybe I'll add that now and just fallback to `findDOMNode` and log a DEV warning if people don't forward the ref properly

Reply:
Good plan

Reply:
This conversation was not marked as resolved, is react-window DynamicSizeList compatible with concurrent mode? I'm about to enable it.